common/headers_test.go

```diff
@@ -0,0 +1,203 @@
+package common_test
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/zeta-chain/zetacore/common"
+
+	"testing"
+)
+
+const numHeadersToTest = 100
+
+func TestTrueBitcoinHeader(t *testing.T) {
+	blocks := LoadTestBlocks()
+
+	for _, b := range blocks.Blocks {
+		// Deserialize the header bytes from base64
+		headerBytes, err := base64.StdEncoding.DecodeString(b.HeaderBase64)
+		if err != nil {
+			t.Error(err)
+		}
```

Comment:
I think we could use `github.com/stretchr/testify/require` like we use in tests throughout the codebase 
```
require.Error(t, err)
```

Reply:
I'll use `require.NoError(t, err)` to avoid log prints.